src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/ACHPayee.xml

```diff
@@ -33,6 +55,20 @@
     <property name="fieldOverrides">
       <list>
         <!-- Add "principalName" to the ACHPayee lookup input fields. -->
+
+        <bean parent="FieldOverrideForListElementInsert">
+          <property name="propertyName" value="displayAttributeDefinitions" />

```

Comment:
These new field override beans need adjustments so that they line up with the XML comments. ("formAttributeDefinitions" represents the search criteria fields; "displayAttributeDefinitions" represents the search results fields. Also, "ACHPayee-entityId" is in the search criteria, whereas "DisbursementPayee-payeeIdNumber" is in the search results.)

Reply:
update